routing/base/routing_result.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "routing/base/astar_weight.hpp"
+
+#include <vector>
+
+namespace routing
+{
+template <typename TVertexType, typename TWeightType>
+struct RoutingResult
+{
+  std::vector<TVertexType> m_path;
+  TWeightType m_distance;
+  RoutingResult() : m_distance(GetAStarWeightZero<TWeightType>()) {}
+  void Clear()
```

Comment:
Empty line between Clear() and ctor, please.

Reply:
done but clang-format does not like it